torch/csrc/jit/ir.h

```diff
@@ -565,6 +573,12 @@ friend struct Node;
     return n;
   }
 
+  // In some cases, the compiler is not able do distinguish between
+  // overloads of `create` function. This helps.
```

Comment:
LOL, I guess this is because strings can default construct into ArrayRef. It might be worth rearranging the API in a bigger way to solve this; perhaps setting the pretty name via a setter exclusively.